Cake.Recipe/Content/build.cake

```diff
@@ -180,14 +180,12 @@ BuildParameters.Tasks.BuildTask = Task("Build")
                 .WithProperty("TreatWarningsAsErrors", BuildParameters.TreatWarningsAsErrors.ToString())
                 .WithTarget("Build")
                 .SetMaxCpuCount(ToolSettings.MaxCpuCount)
-                .SetConfiguration(BuildParameters.Configuration)
-                .WithLogger(
-                    Context.Tools.Resolve("MSBuild.ExtensionPack.Loggers.dll").FullPath,
-                    "XmlFileLogger",
-                    string.Format(
-                        "logfile=\"{0}\";invalidCharReplacement=_;verbosity=Detailed;encoding=UTF-8",
-                        BuildParameters.Paths.Files.BuildLogFilePath)
-                );
+                .SetConfiguration(BuildParameters.Configuration);
+
+            msbuildSettings.ArgumentCustomization = args =>
+                args.Append(string.Format("/logger:BinaryLogger,\"{0}\";\"{1}\"",
+                    context.Tools.Resolve("Cake.Issues.MsBuild*/**/StructuredLogger.dll"),
```

Comment:
The reason you used `StructuredLogger.dll` instead of `\bl` is to have control about version of binary log?

Reply:
Yup, correct. I want to ensure that the same format generated by the logger is supported by Cake.Issues.MsBuild. So what better way than to use the structured logger that adding uses :smile: